docs/core/diagnostics/event-counter-perf.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Measure performance using EventCounters in .NET Core
+description: Installing and using the dotnet-trace command-line tool.
+ms.date: 07/22/2020
+---
+
+# Tutorial: Measure performance using EventCounters in .NET Core
+
+**This article applies to: ✔️** .NET Core 3.0 SDK and later versions
+
+While an <xref:System.Diagnostics.Tracing.EventSource> is fast, logging too many events for frequent events is still a performance consideration. In this tutorial, you'll learn how an <xref:System.Diagnostics.Tracing.EventCounter> can be used to measure performance of a high frequency of events.
```

Comment:
This text assumes customers are thinking "I am logging so many EventSource events that performance is bad, how can I go faster?" My educated guess is that this is not the frame of reference most customers would relate to and instead the common mental frames will be:
a)  I would like to monitor .NET built-in metrics for my app, similar to what I could with PerfMon. How can I do that?
b) I would like to add my own metrics and monitor them, how can I do that?

For the first case a tutorial like the first part of [this one](https://github.com/dotnet/diagnostics/blob/master/documentation/tutorial/app_is_leaking_memory_eventual_crash.md) might be more useful.
For the second one that could be this doc, but I suggest rephrasing the intro and switch from PerfView to dotnet-counters.

Reply:
@IEvangelist - I noticed you had been doing some minor tweaking of the wording which wasn't what I had expected. I was thinking to delete line 11-15 inclusive and replace it with a new intro that matches up better with the frame of reference I expect customers looking at this page would have.
I wasn't sure if you were making different changes because you disagreed with my suggestion above (which is fine and I'd be glad to talk about) or I just didn't do a good job explaining.